chapter_12_organising_test_files.asciidoc

```diff
@@ -205,7 +199,6 @@ class ItemValidationTest(FunctionalTest):
 ----
 ====
 
-// DAVID: I also have check_for_row_in_list_table as a method - maybe I should've removed it?
```

Comment:
Yes, it was in Chapter 06, right above "Example 4. functional_tests/tests.py (ch06l004)". We renamed this function.